### DISCUSSION TITLE
Can you return a custom response on event (`EventNoFreeWorkers`, `no free workers in the pool, wait timeout exceed`)

### DISCUSSION THREAD

**Discussion by aljaazm** [roadrunner-server/roadrunner#1647] [Q&A] [answered]
At the moment, when all workers are busy and the wait timeout is exceeded, a 500 Server Error response without body is returned. I'd like to change the response for that specific error to a 503 Service Unavailable and add a json body. Is that possible?

**Comment by rustatian** [comment]
Hey @aljaazm 👋🏻 Sure, just set the needed code here: [link](https://github.com/roadrunner-server/roadrunner/blob/master/.rr.yaml#L626) 😃

**Reply by rustatian** [reply, marked as answer]
> and add a json body

Unfortunately it's not possible to return a body with this kind of error at the moment. However, you can create a feature request and if the community supports this ticker, I'd be happy to implement it.
